portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro

```diff
@@ -243,10 +243,11 @@ definition {
 	macro editDateCalendar {
 		Click(locator1 = "FormFields#DATE_CALENDAR");
 
-		Select(
-			key_valueName = "year",
-			locator1 = "FormFields#DATE_CALENDAR_FIELD",
-			value1 = ${valueYear});
+		Click(locator1 = "FormFields#YEAR_CALENDAR_FIELD");
+
+		Click(
+			key_optionYear = ${valueYear},
+			locator1 = "FormFields#YEAR_CALENDAR_OPTION");
```

Comment:
The behavior has been altered as described in the Bug  - [LPS-189643](https://liferay.atlassian.net/browse/LPS-189643).
All the tests that use this macro have been tested and are on the ticket.